templates/UmbracoPackage/UmbracoPackage.csproj

```diff
@@ -1,16 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
     <ContentTargetFolders>.</ContentTargetFolders>
-    <Product>UmbracoPackage</Product>
+    <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">UmbracoPackage</RootNamespace>
+  </PropertyGroup>
+
+  <PropertyGroup>
     <PackageId>UmbracoPackage</PackageId>
+    <Product>UmbracoPackage</Product>
     <Title>UmbracoPackage</Title>
     <Description>...</Description>
     <PackageTags>umbraco plugin package</PackageTags>
-    <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">UmbracoPackage</RootNamespace>
-    <ManagePackageVersionsCentrally>false</ManagePackageVersionsCentrally>
```

Comment:
This was part of the distributed project template (for packages and new Umbraco projects)!

Reply:
I guess the rootnamespace should be. Also the other one did not do any harm, due to the fact we was not using centrallized packages for those

Reply:
The root namespace is still there, just moved to a different property group 😉 We don't know whether the template is used to create a new project within a solution that already uses CPM and we also don't want the project to opt-out if you enable it later either!